landing/src/app.css

```
@@ -585,3 +588,137 @@ pre,
+  .body-text {
+    font-size: 0.9375rem;
+  }
+}
+
+/* --- Mobile code block styling --- */
+@media (max-width: 767px) {
+  pre,
+  code {
+    font-size: 0.7rem !important;
+    white-space: pre-wrap !important;
+    word-break: break-word;
```

Stylelint (17.4.0)

[error] Unexpected deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)